packages/node_modules/@webex/plugin-meetings/test/unit/spec/peerconnection-manager/index.js

```diff
@@ -19,6 +19,16 @@ describe('Peerconnection Manager', () => {
         onicecandidateerror: null,
         localDescription: {sdp}
       };
+
+      Object.defineProperty(global.window, 'RTCSessionDescription', {
```

Comment:
For fixing the `TypeError: _window.default.RTCSessionDescription is not a constructor` unit error , I was looking for this change, I checked the PR diff and get to know about it. 

Thanks for doing this..